components/Chat/ChatSettings/ChatSettingsPanel/ChatGeneralSettingsPanel.vue

```diff
@@ -42,20 +43,22 @@
       >
         <template #label>
           <div>
-            <span>Receive @'s in local</span>
+            <span>Receive <b>@mentions</b> in local</span>
             <v-tooltip top color="blue-grey darken-4">
               <template #activator="{ on: tooltip }">
                 <v-btn v-on="{ ...tooltip }" class="ml-1" x-small icon>
                   <v-icon color="grey">help_outline</v-icon>
                 </v-btn>
               </template>
-              <span>Show chat messages that you are @tagged in<br>
-                 from every channels</span>
+              <span>Show chat messages that you are <b>@tagged</b> in<br>
+                 from every channel</span>
             </v-tooltip>
           </div>
         </template>
       </v-switch>
 
+      <v-divider id="divider" />
```

Comment:
a class would be better than an id. This also seems to just set top / bottom margins, is `my-1` or other general `my-#` not sufficient?

Reply:
`my-#` didn't seem to work, neither did the CSS ruling when `divider` was a class

Reply:
> Is any general my-# not sufficient?
The idea was to have a single rule that does everything, instead of copy-pasting classes. It makes it more obvious what everything does when reading later on.

Reply:
even if you did `#chat-settings > .divider` ? that should be a high enough selector to override it's values. I just don't like having a duplicated ID in the DOM.

Reply:
Not with the entire selector. Let me try that

Reply:
Seems to work with the full selector. Or maybe it was a Nuxt thing. Committing changes...